app/screens/move-money-screen/move-money-screen.tsx

```diff
@@ -178,14 +178,12 @@ export const MoveMoneyScreenDataInjected: ScreenType = ({
 
   function isUpdateAvailableOrRequired({ buildParameters }) {
     try {
-      const {
-        minBuildNumberAndroid,
-        minBuildNumberIos,
-        lastBuildNumberAndroid,
-        lastBuildNumberIos,
-      } = buildParameters
-      const minBuildNumber = isIos ? minBuildNumberIos : minBuildNumberAndroid
-      const lastBuildNumber = isIos ? lastBuildNumberIos : lastBuildNumberAndroid
+      const minBuildNumber = isIos
+        ? buildParameters?.minBuildNumberIos
+        : buildParameters?.minBuildNumberAndroid
+      const lastBuildNumber = isIos
+        ? buildParameters?.lastBuildNumberIos
+        : buildParameters?.lastBuildNumberAndroid
```

Comment:
This looks like V1. Did we miss this one?

Reply:
Yeah looks like it.  I missed that too.  Do we have a v2 endpoint for these?